ROLLING.md

```diff
@@ -1,17 +1,18 @@
 # Rolling Playwright-dotnet to the latest Playwright driver
 
 ## Pre-requisites
-* remove old gen files: `find src/Playwright | grep Generated | xargs rm`
-* install dotnet with installer: https://dotnet.microsoft.com/download/dotnet/5.0
-* install dotnet format tool: `dotnet tool install -g dotnet-format`
+
+* install .NET SDK 5 with installer: https://dotnet.microsoft.com/download/dotnet/5.0
 * install [PowerShell](https://github.com/PowerShell/PowerShell)
 
 ## Rolling
-* obtain the commit hash (long form) to where we're rolling to
-* in the repo folder, run
 
-    ```powershell
-    .\Build.ps1 roll <commit hash>
-    ```
+1. Checkout the upstream release branch of the `playwright` repository.
+1. Checkout the upstream release branch of the `playwright-dotnet` repository.
+1. Pick the latest driver from GitHub Action where you want to roll to. Usually from the Node.js release.
+
+```powershell
+./build.ps1 roll <driver-version>
+```
 
-This will complete the entire process including replacing the version number and downloading drivers.
\ No newline at end of file
+This will complete the entire process including replacing the version number and downloading drivers.
```

Comment:
I hope this does not update the version number of playwright-dotnet itself? Let's be clear here.

Reply:
It does not anymore, this got removed.